src/Integration.UnitTests/LocalServices/UnboundProjectFinderTests.cs

```diff
@@ -65,17 +68,17 @@ public void GetUnboundProjects_ValidSolution_ReturnsProjectsThatRequireBinding(i
 
             foreach (var boundProject in boundProjects)
             {
-                testConfig.SetupProjectBindingRequired(boundProject, true);
+                testConfig.SetupProjectBindingRequired(boundProject, false);
             }
             foreach (var boundProject in unboundProjects)
             {
-                testConfig.SetupProjectBindingRequired(boundProject, false);
+                testConfig.SetupProjectBindingRequired(boundProject, true);
             }
 
             var testSubject = testConfig.CreateTestSubject();
 
             var result = testSubject.GetUnboundProjects();
-            result.Should().AllBeEquivalentTo(unboundProjects);
```

Comment:
I'm really surprised the tests didn't fail on this sooner: AllBeEquivalentTo is supposed to receive a single object, not a collection.